publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/expandresources/ExpandedDataEntriesPersistenceHandler.java

```diff
@@ -22,7 +22,7 @@
     extends DestinationsEventBridgeEventHandler<EventReference, EventReference> {
 
     public static final String EXPANDED_ENTRY_PERSISTED_EVENT_TOPIC = "PublicationService.ExpandedEntry.Persisted";
-    public static final String PERSISTED_ENTRIES_BUCKET = ENVIRONMENT.readEnv("PERSISTED_ENTRIES_BUCKET");
```

Comment:
moved to static class PersistenceConfig. This class will be moved to the `persistence` package in later PR